reactioncommerce/reaction
# integrate catalyst ConfirmDialog component from new Catalyst package

Resolves #5359
Impact: minor
Type: feature

## Summary
This PR adds the new [Catalyst](https://github.com/reactioncommerce/catalyst) package and integrates the `ConfirmDialog` component wherever it was used. Further, the MUI theme is now imported from the [Catalyst](https://github.com/reactioncommerce/catalyst) and the existing MUI theme in Reaction has been removed.

## Testing
1. Verify the confirm dialog renders correctly in the Navigation page, attempt to remove delete a navigation item and remove an item from menu.
2. Verify confirm dialog render correctly when attempting to archive a product.
